chromium_src/chrome/browser/ui/autofill/payments/save_card_bubble_controller_impl.cc

```diff
@@ -0,0 +1,11 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "../../../../../../../chrome/browser/ui/autofill/payments/save_card_bubble_controller_impl.h"
+
+#define BRAVE_SHORT_CIRCUIT_TO_FALSE \
+return false;
```

Comment:
please add an empty line here so multiline macro end doesn't get mixed up with `#include`

Reply:
Fixed.

Reply:
@mkarolin can you open a ticket to look at using the sync settings instead of this patch? We'll just keep the patch for now